src/Parser/Warrior/Fury/Modules/BuffDebuff/Juggernaut.js

```diff
@@ -27,12 +28,26 @@ class juggernautReset extends Analyzer {
       this.stacksMax = event.newStacks;
     }
   }
+  
+  get suggestionThresholds() {
+    return {
+      isGreaterThan: {
+        minor: 0,
+      },
+      style: 'percentage',
+    };
```

Comment:
Something I noticed when implementing my own checklist yesterday was that you'll need all 3 levels of severity of the issue for it to work properly and not simply be a white bar, so might want to look into that

Reply:
I have yet to work on the checklist, so I wouldn't know. Can the severity levels have the same value? In general Juggernaut stacks should never drop - but seeing as there may be adds it could be unavoidable. Having Juggernaut drop on a Patchwerk boss, however, is a major issue.

Reply:
Well all of these things ``suggestionThresholds`` you're making are strictly speaking only a thing because of the checklist as that's where they're used. 
I never checked if they were the same, right now it's PERMANENTLY a minor issue, it'll never go to average or major. 
What you can do is minor: 0.5
average: 0.7
major: 0.9
Then if it becomes 1, it's a major issue
Or maybe like so:
minor: 0.5 (or 0, whatever)
average: 0.9
major: 1.1
In which case if it drops twice, it's a major issue, but otherwise average if it drops (then in the suggestion include a note on AoE about it)

Reply:
I made this kind of suggestionThreshold due to @MartijnHols's suggestion on my previous PR #982. I do, however, agree with your interpretation that it would permanently be a minor issue, but my testing showed it as a major issue regardless of it only having a minor threshold. I'm guessing it's because there is only one value, and seeing as we're below that (and there is no additional average or major thresholds) it defaults to major. I'll get it fixed for my next commits.